### DISCUSSION TITLE
A linter for sceptre templates

### DISCUSSION THREAD

**Discussion by zaro0508** [cloudreach/sceptre#925] [Ideas]
We are developing a set of [pre-commit](https://pre-commit.com/) hooks to help validate our Sceptre templates.  Please take a look and let us know what you think, https://github.com/Sage-Bionetworks-IT/pre-commit-provisioner

**Comment by craighurley** [comment]
This is great.  Can we migrate to the sceptre org?

**Comment by zaro0508** [comment]
done @craighurley, the project is at https://github.com/Sceptre/sceptrelint
